engine/apps/slack/views.py

```diff
@@ -545,7 +545,7 @@ def post(self, request):
         if slack_team_identity is not None:
             clean_slack_integration_leftovers.apply_async((organization.pk,))
             if settings.FEATURE_MULTIREGION_ENABLED:
-                delete_slack_connector_async.apply_async((slack_team_identity.slack_id,))
+                delete_slack_connector(str(organization.uuid))
```

Comment:
💡 may be uuid conversion into `delete_slack_connector`

Reply:
I don't want to specify, that delete_slack_connector (and all other *_connector functions) should receive oncall_org_id with uuid type.